crates/router/src/core/revenue_recovery/types.rs

```diff
@@ -143,6 +144,18 @@ impl RevenueRecoveryPaymentsAttemptStatus {
                     );
                 };
 
+                // update the status of token in redis
+
+                // Update CALCULATE_WORKFLOW to complete status on payment success
+                if let Err(e) = update_calculate_workflow_on_success(state, payment_intent).await {
```

Comment:
Why we are updating the calculate workflow on success of a payment?, on completion of calculate workflow only a execute task itself created